maestro/flows/login.yaml

```
@@ -0,0 +1,59 @@
+    - tap:
+        id: "password_visibility_toggle"
+    - assertVisible:
+        id: "password_field"
+        text: "testpassword"
+    - takeScreenshot: "password_visible"
+    - tap:
+        id: "password_visibility_toggle"
+    - assertNotVisible:
+        id: "password_field"
+        text: "testpassword"
+    - takeScreenshot: "password_hidden"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)